apps/devhub/tests/test_views.py

```diff
@@ -1142,11 +1144,11 @@ def test_step1_submit(self):
     def test_read_dev_agreement_set(self):
         """Store current date when the user agrees with the user agreement."""
```

Comment:
the currently signed in user will have their `read_dev_agreement` [already configured](https://github.com/Nolski/olympia/blob/signing-api-agreement/apps/amo/fixtures/base/addon_3615.json#L258). For the purpose of this test, you should initialize it to None before any request is made, like `self.user.update(read_dev_agreement=None)`